executor/load_data.go

```diff
@@ -352,10 +355,15 @@ func escapeChar(c byte) byte {
 		return '\t'
 	case 'Z':
 		return 26
+	case 'N':
+		f.maybeNull = true
+		return c
 	case '\\':
-		return '\\'
+		return c
+	default:
+		log.Warnf("illegal escaped character '%c'!", c)
```

Comment:
This warning is misleading, it's not illegal.

Reply:
Any suggestions?